swap/src/asb/event_loop.rs

```diff
@@ -479,16 +479,34 @@ where
         // use unlocked monero balance for quote
         let xmr_balance = Amount::from_piconero(balance.unlocked_balance);
 
-        let max_bitcoin_for_monero =
-            xmr_balance
-                .max_bitcoin_for_price(ask_price)
-                .ok_or_else(|| {
-                    anyhow!(
-                        "Bitcoin price ({}) x Monero ({}) overflow",
-                        ask_price,
-                        xmr_balance
-                    )
-                })?;
+        // From our full balance we need to subtract any Monero that is 'reserved' for ongoing swaps
+        // (where the Bitcoin has been (or is being) locked but we haven't sent the Monero yet).
+
```

Comment:
Consider adding a TODO comment that this is "naive" to some degree due to the 20min lock of outputs / utxo issue